cibuildwheel/__main__.py

```diff
@@ -300,7 +308,7 @@ def main() -> None:
         repair_command=repair_command,
         environment=environment,
         dependency_constraints=dependency_constraints,
-        manylinux_images=manylinux_images,
+        manylinux_images=manylinux_images or None,
```

Comment:
Note for later: some tests assume this is none, but it probably is simpler type-wise just to leave it an empty set. Potential cleanup unrelated to this PR, though.